src/app/story/storyrow/storyrow.component.html

```diff
@@ -1,11 +1,16 @@
 <div class="node-uid-column story-row-column">
-  <div class="node-display-name">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
+  <div class="node-display-name lengthened">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
+  <div class="node-display-name shortened">{{ node.display_name | nodeDisplayName | truncate: 15 }}</div>
   <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">
     {{ node.timeline_timestamp | date: 'MMM d, y, HH:mm:ss':'+0000' }} UTC
```

Comment:
Could you hide the `UTC` in mobile mode? That would make the dates lineup better.